src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
@ranjitjhala from the refined definition `instance ToProve a (Proof -> b) where` I want to create the dictionary variables `$fOptEqa(->)` and `$fOptEqa(->)0` to get the variable names ghc creates.